### PR TITLE
Deprecation for 0.16 (#6581)

### PR DESCRIPTION
**Pull request by jsexauer**
These are the deprecations that are to be removed in 0.16, per #6581, except for the deprecations around the boxplot function.

### CODE REVIEW

File: doc/source/whatsnew/v0.16.0.txt [hunk 1]
```diff
@@ -217,6 +217,12 @@ Deprecations
 
 .. _whatsnew_0160.deprecations:
 
+- ``DataFrame.pivot_table`` and ``crosstab``'s `rows` and `cols` keyword arguments were removed in favor
+  of `index` and `columns` (:issue:`6581`)
```
Comment (jreback) [inline doc/source/whatsnew/v0.16.0.txt]: need double backquotes around rows/cols (and index/columns)

Comment (jsexauer) [inline doc/source/whatsnew/v0.16.0.txt]: @jreback also for `covert_dummies`, `value_range`, etc...?

Comment (jreback) [inline doc/source/whatsnew/v0.16.0.txt]: yep anything you want to make look like code (basically highlite it)

File: doc/source/whatsnew/v0.16.0.txt [hunk 2]
```diff
@@ -217,6 +217,12 @@ Deprecations
 
 .. _whatsnew_0160.deprecations:
 
+- ``DataFrame.pivot_table`` and ``crosstab``'s ``rows`` and ``cols`` keyword arguments were removed in favor
+  of ``index`` and ``columns`` (:issue:`6581`)
+- ``DataFrame.to_excel`` and ``DataFrame.to_csv`` ``cols`` keyword argument was removed in favor of ``columns`` (:issue:`6581`)
+- Removed ``covert_dummies`` in favor of ``get_dummies`` (:issue:`6581`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.16.0.txt]: covert -> convert

**Comment by jreback** [comment]
@shoyer @jorisvandenbossche look ok to you? (other than the typo)

**Comment by jorisvandenbossche** [comment]
yep, looks ok!

**Comment by jreback** [comment]
@jsexauer thanks!

**Pull request merged**